utils/generate-coverage

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+set -eu -o pipefail
+
+HARE_BASE_DIR=/opt/seagate/cortx/hare
+PYTHONPATH="$HARE_BASE_DIR/lib/python3.6/site-packages"
+PYTHONPATH="$HARE_BASE_DIR/lib64/python3.6/site-packages:$PYTHONPATH"
+export PYTHONPATH
+		
+PATH="$HARE_BASE_DIR/bin:$PATH"
+export PATH
+
+activate () {
+    . .py3venv/bin/activate
```

Comment:
Please note that this works when the script is invoked from the source folder. In the environment where Hare is installed from RPM, this command will fail (RPM doesn't bring the activation scripts, it ships dependencies only).

Is this script supposed to be run after RPM installation?

Reply:
Yes, For RPM installed hare it wont work. Added Note into the readme as well as into the wrapper script also.